builder-config/builder-template.yaml

```diff
@@ -0,0 +1,10 @@
+# Build pipeline template, for use during development and testing
```

Comment:
This file is almost identical to `java.yaml`. To avoid code duplication, can we do some `sed` magic instead in the `run_pipeline.sh` and get rid of this file?